### PR TITLE
Added NLLB script

### PR DESCRIPTION
**Pull request by Mofetoluwa**

### CODE REVIEW

**Comment by ToluClassics** [comment]
Thanks!

File: translation_script/nllb_translate.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    tokenizer = AutoTokenizer.from_pretrained("facebook/nllb-200-distilled-600M", 
+        src_lang=flores_codes[args.lang])
+    model = AutoModelForSeq2SeqLM.from_pretrained("facebook/nllb-200-distilled-600M")
+
+    inputs = tokenizer(queries, padding=True, truncation=True, return_tensors="pt")
+
+    print("Translating Queries...")
+    if args.lang == 'fon' or args.lang == 'wolof':
+        translated_tokens = model.generate(
+            **inputs, forced_bos_token_id=tokenizer.lang_code_to_id["fra_Latn"], max_length=30
```
Comment (ToluClassics) [inline translation_script/nllb_translate.py:73]: I'm just making sure that max_length here does not affect the generations? and does this produce only one translation per question which is what we want

Comment (Mofetoluwa) [reply translation_script/nllb_translate.py:73]: Yeah it's one translation per question. 
Hmm, I left it at 30 coz I felt the questions might not have up to 30 tokens? What value can I change it to

Comment (ToluClassics) [reply translation_script/nllb_translate.py:73]: 30 is probably fine, can we test this on one of the language files, probably the one you have access to; if  it's okay let's  merge

Comment (Mofetoluwa) [reply translation_script/nllb_translate.py:73]: Yup...can I send the result here

Comment (ToluClassics) [reply translation_script/nllb_translate.py:73]: Naah, if you think it's okay we can merge

**Review by ToluClassics — APPROVED** [review]

File: translation_script/nllb_translate.py (hunk 1 above)
Comment (Mofetoluwa) [reply translation_script/nllb_translate.py:73]: Yeah it looks fine

**Pull request merged**